tests/integration/testdata/buildcmd/base-dir/template/template.yaml

```diff
@@ -0,0 +1,17 @@
+AWSTemplateFormatVersion : '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: A hello world application.
+
+Resources:
+  FunctionA:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: main_a.handler
+      Runtime: python3.7
+      CodeUri: FunctionA
+      Timeout: 600
+
+  ChildStackX:
+    Type: AWS::Serverless::Application
+    Properties:
+      Location: ../ChildStackX/template.yaml
```

Comment:
This will combine with provided base dir right?

Reply:
no this will not

Reply:
However if a base dir is not provided, we just resolve with location of the template. (which is not this particular test, but asking to help grok through the test)

Reply:
yes, whether base dir is provided or not, this `Location` is resolved according to its parent template path.

Reply:
on offline conversation: applies only to codeuri and contenturi.